scrapers/distgit.py

```diff
@@ -143,6 +146,11 @@ def update_neo4j(self, results):
             commit.branches.connect(branch)
             commit.repos.connect(repo)
 
+            if repo_info['parent']:
+                parent_commit = DistGitCommit.get_or_create({'hash_': repo_info['parent']})[0]
+                commit.parents.connect(parent_commit)
+                parent_commit.children.connect(commit)
```

Comment:
I thought yesterday we decided on not having child connections since they can be different for different branches. Did something change ?

Reply:
Yes, after talking with @ralphbean we decided to do it since it's easy to implement with the current solution.

Reply:
ah, I see :)